src/ImageSharp/Formats/Jpeg/Components/FastFloatingPointDCT.Intrinsic.cs

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+#if SUPPORTS_RUNTIME_INTRINSICS
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace SixLabors.ImageSharp.Formats.Jpeg.Components
+{
+    internal static partial class FastFloatingPointDCT
+    {
+#pragma warning disable SA1310, SA1311, IDE1006 // naming rules violation warnings
+        private static readonly Vector256<float> mm256_F_0_7071 = Vector256.Create(0.707106781f);
```

Comment:
While this is true for .net 5, .net core 3 seems to struggle with this:

.net 5
|           Method |     Mean |     Error |    StdDev |
|----------------- |---------:|----------:|----------:|
|    'Static load' | 7.561 ns | 0.0445 ns | 0.0416 ns |
| 'Create inplace' | 7.786 ns | 0.0211 ns | 0.0187 ns |

.net core 3.1
|           Method |     Mean |     Error |    StdDev |
|----------------- |---------:|----------:|----------:|
|    'Static load' | 7.316 ns | 0.0280 ns | 0.0219 ns |
| 'Create inplace' | 9.631 ns | 0.0269 ns | 0.0239 ns |

I'm not sure if we should sacrifice .net core performance in favor of .net 5. Especially considering that static loads are as fast on .net 5 as inline `Create`'s.

*tested on 1D FDCT routine

Reply:
Above is especially important for Zig-Zag method which run even slower than scalar implementation if all masks would be created as `VectorXYZ.Create` - this was tested long time ago but I've deleted benchmark results :(

Reply:
> we should sacrifice .net core performance in favor of .net 5

Good point, that I missed. Maybe we should keep the static readonly code for good perf on all targets, log an issue to address this when .NET Core 3.1 gets out of support on [3rd Dec 2022](https://dotnet.microsoft.com/platform/support/policy)?

There are some (theoretical) benefits of using "inline vector create":
* faster startup -- no static initialization
* code with loops will be Tier-1 compiled (when not configured otherwise), so the static initialization check won't be eliminated -- this isn't a problem with the inline vector create / or when code with loops will be tiered compiled normally (but almost no app configures this setting, or do they?)
* smaller produced machine code / working set

Reply:
Separate 'future' issue is a good idea. I totally agree with you about `VectorXYZ.Create`.

Reply:
Filed https://github.com/SixLabors/ImageSharp/issues/1762 for this.